src/components/calcite-dropdown-item/calcite-dropdown-item.scss

```diff
@@ -129,18 +137,68 @@
   padding-right: $baseline/1.5;
 }
 
+// single select "icon"
+:host-context([scale="s"]):before {
+  left: theme("spacing.2");
+}
+:host-context([scale="m"]):before {
+  left: theme("spacing.3");
+}
+:host-context([scale="l"]):before {
+  left: theme("spacing.4");
+}
+
+// single select "icon" RTL
+:host-context([dir="rtl"]):before {
+  left: unset;
+}
+
+:host-context([dir="rtl"][scale="s"]):before {
+  right: theme("spacing.2");
+}
+:host-context([dir="rtl"][scale="m"]):before {
+  right: theme("spacing.3");
+}
+:host-context([dir="rtl"][scale="l"]):before {
+  right: theme("spacing.4");
+}
+
 // multi select check icon
 :host .dropdown-item-check-icon {
```

Comment:
Nitpicky - on Large scale - we should use a "m" icon vs. the current "s" - cc @bstifle

Reply:
Good idea. Being concentrated on font-size, I didn't catch that.

Reply:
Thanks. I know it's out of scope but it's... scope adjacent, lol.